alert-database/src/main/java/com/synopsys/integration/alert/database/certificates/ClientCertificateRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.synopsys.integration.alert.database.certificates;
+
+import java.util.UUID;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ClientCertificateRepository extends JpaRepository<ClientCertificateEntity, UUID> {
```

Comment:
We likely want some helper JPA methods here. Such as findByAlias, existsByAlias, and deleteByAlias. Similar to what we've done in SAMLConfigurationRepository. This way we can move and simplify some of the logic out of the accessor and into JPA.